redaxo/src/core/lib/sql/sql.php

```diff
@@ -1260,25 +1260,25 @@ function ($matches) use ($params, &$i) {
     /**
      * Setzt eine Spalte auf den naechst moeglich auto_increment Wert.
      *
-     * @param string $field    Name der Spalte
-     * @param int    $start_id
+     * @param string $colName Name der Spalte
+     * @param int    $startId
      *
      * @throws rex_sql_exception
      *
      * @return int
      */
-    public function setNewId($field, $start_id = 0)
+    public function setNewId($colName, $startId = 0)
```

Comment:
`$colName` schien mir zu überwiegen in der Datei, deswegen habe ich das genommen.
Wobei ich mir aber unsicher bin, ob ich nicht `$field` eigentlich besser fände.

Reply:
bin ich leidenschaftslos. finde beide ok. nimm was dir besser gefällt